mini-project-1/products.json

```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
     "productImage": "assets/product_images/cafe1.png",
     "discount": "nodiscount",
```

Comment:
What about changing this k-v pair from `"discount": "nodiscount"` to: `onDiscount: false`.
When we have only 2 options, it's better to keep it simple and use a boolean instead.